js/private/test/image/custom_owner_test_package_store_1p.bzlmod.listing

```diff
@@ -1,13 +1,13 @@
-drwxr-xr-x  0 100    0           0 Jan  1  1970 app
-drwxr-xr-x  0 100    0           0 Jan  1  1970 app/js
-drwxr-xr-x  0 100    0           0 Jan  1  1970 app/js/private
-drwxr-xr-x  0 100    0           0 Jan  1  1970 app/js/private/test
-drwxr-xr-x  0 100    0           0 Jan  1  1970 app/js/private/test/image
-drwxr-xr-x  0 100    0           0 Jan  1  1970 app/js/private/test/image/bin.runfiles
-drwxr-xr-x  0 100    0           0 Jan  1  1970 app/js/private/test/image/bin.runfiles/_main
-drwxr-xr-x  0 100    0           0 Jan  1  1970 app/js/private/test/image/bin.runfiles/_main/node_modules
-drwxr-xr-x  0 100    0           0 Jan  1  1970 app/js/private/test/image/bin.runfiles/_main/node_modules/.aspect_rules_js
-drwxr-xr-x  0 100    0           0 Jan  1  1970 app/js/private/test/image/bin.runfiles/_main/node_modules/.aspect_rules_js/@mycorp+pkg-d@0.0.0
-drwxr-xr-x  0 100    0           0 Jan  1  1970 app/js/private/test/image/bin.runfiles/_main/node_modules/.aspect_rules_js/@mycorp+pkg-d@0.0.0/node_modules
-drwxr-xr-x  0 100    0           0 Jan  1  1970 app/js/private/test/image/bin.runfiles/_main/node_modules/.aspect_rules_js/@mycorp+pkg-d@0.0.0/node_modules/@mycorp
-lrwxrwxr-x  0 100    0           0 Jan  1  1970 app/js/private/test/image/bin.runfiles/_main/node_modules/.aspect_rules_js/@mycorp+pkg-d@0.0.0/node_modules/@mycorp/pkg-d -> ../../../../../examples/npm_package/packages/pkg_d
+drwxr-xr-x  0 100    0           0 Jan  1  1970 ./app/
```

Comment:
Is the `./` + `/` change intentional? Why?

Reply:
this something bsdtar does by default. i guess to maintain compatibility, i should revert it back to original.  `./app` and `app` mean the same thing for overlayfs

Reply:
on second thought, i think i will leave this as is, unless somebody complains.